bottles/frontend/ui/details-preferences.ui

```diff
@@ -59,7 +59,7 @@
                     <object class="AdwActionRow" id="row_nvapi">
                         <property name="activatable-widget">switch_nvapi</property>
                         <property name="title" translatable="yes">DLSS (DXVK-NVAPI)</property>
-                        <property name="subtitle" translatable="yes">Provide DLSS support if available and Nvidia's NVAPI.</property>
+                        <property name="subtitle" translatable="yes">Provide DLSS support if available, and Nvidia's NVAPI.</property>
```

Comment:
I would rephrase this to be `Provide DLSS and Nvidia NVAPI support if available`